serverdensity/sd-agent
# Fix virtualenv paths

#### What's this PR do?
Ensure packages virtualenv has the correct paths configured

#### Where should the reviewer start?
debian/rules is a good place to start with

#### How should this be manually tested?
Build and install this package, check that using agent's pip destination path for new installed packages is correct.

#### What are the relevant links or services impact?
Agent

#### Infrastructure impact
None

#### Documentation impact
None

#### Who should review or be notified about this?
* Reviewers
@serverdensity/backend-engineering
